components/Projects/ProjectCard/DiscordButton.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+export interface DiscordButtonProps {
+  link: string;
+}
+
+export default function DiscordButton({ link }: DiscordButtonProps) {
```

Comment:
We already have `DiscordLink` in `@/components/Common/DiscordLink`

Reply:
Yes, makes sense to use the Common DiscordLink